### PR TITLE
Tiltfile should not deploy the on-disk workload

### PR DESCRIPTION
**Pull request by paulcwarren**
- otherwise it will override the workload just deployed by tanzu cli 

[TD-240](https://jira.eng.vmware.com/browse/TD-240)"

### CODE REVIEW

**Comment by lisamburns** [comment]
"otherwise it will override the workload just deployed by tanzu cli"
@paulcwarren this sounds like a bug but I'm not sure how to reproduce it, or what the weird behavior you're seeing is.

**Comment by paulcwarren** [comment]
@hnandiwada this one is of interest to you

File: Tiltfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
+load('.tanzu/tanzu_tilt_extensions.py', 'tanzu_k8s_yaml')
+
 custom_build('harbor-repo.vmware.com/tanzu_desktop/sample-app-java', 
-    "tanzu apps workload apply -f config/workload.yaml --local-path=. --yes && \
+    "tanzu apps workload apply -f config/workload.yaml --live-update --local-path=. --source-image=harbor-repo.vmware.com/tanzu_desktop/sample-app-java-souce --yes && \
```
Comment (hnandiwada) [inline Tiltfile:4]: not sure if this was working before but there is a type in sample-app-java-source

**Review by hnandiwada — CHANGES_REQUESTED** [review]

File: Tiltfile (hunk 1 above)
Comment (paulcwarren) [reply Tiltfile:4]: Good spot

**Comment by paulcwarren** [comment]
> "otherwise it will override the workload just deployed by tanzu cli"
> @paulcwarren this sounds like a bug but I'm not sure how to reproduce it, or what the weird behavior you're seeing is.

There are more details in this [tilt issue](https://github.com/Tilt-Dev-Customers/vmware/issues/1).

To repo:
- update on config/workoad.yaml with a git source
- run tilt up

Expected behavior:
- you get a workload running the source code with live update enabled

Actual behavior:
- you momentarily get a workload running the source code with live update enabled, quickly replaced by a workload running the git commit and live update disabled

And this is because tilt kubectl applies config/workload overriding what tanzu apps workload apply put out there.

**Review by lisamburns — APPROVED** [review]
👍

**Pull request merged**